integration/solana/simple.spec.ts

```diff
@@ -1,234 +1,229 @@
+        await program.methods.doOps().accounts({ dataAccount: storage.publicKey }).rpc();
 
-        res = await contract.functions.get_values1({ simulate: true });
+        res = await program.methods.getValues1().accounts({ dataAccount: storage.publicKey }).view();
 
-        expect(res.result.toString()).toStrictEqual("1,65263,32767,57896044618658097711785492504343953926634992332820282019728792003956564819966");
+        expect(BigInt(res.return0)).toEqual(1n);
+        expect(Number(res.return1)).toEqual(65263);
+        expect(Number(res.return2)).toEqual(32767);
+        expect(BigInt(res.return3)).toEqual(57896044618658097711785492504343953926634992332820282019728792003956564819966n);
 
-        res = await contract.functions.get_values2({ simulate: true });
+        res = await program.methods.getValues2().accounts({ dataAccount: storage.publicKey }).view();
 
-        expect(res.result.toString()).toStrictEqual(
-            [
-                61200,
-                "",
-                new Uint8Array(Buffer.from("b0ff1e", "hex")),
-                new Uint8Array(Buffer.from("61626364", "hex")),
-                3
-            ].toString()
-        );
+        expect(Number(res.return0)).toEqual(61200);
+        expect(res.return1).toEqual("");
+        expect(res.return2).toEqual(Buffer.from("b0ff1e", "hex"));
+        expect(res.return3).toEqual([0x61, 0x62, 0x63, 0x64]);
+        expect(Number(res.return4.bar3)).toBeDefined();
 
-        await contract.functions.push_zero();
+        await program.methods.pushZero().accounts({ dataAccount: storage.publicKey }).rpc();
 
         let bs = "0xb0ff1e00";
 
         for (let i = 0; i < 20; i++) {
-            res = await contract.functions.get_bs({ simulate: true });
+            res = await program.methods.getBs().accounts({ dataAccount: storage.publicKey }).view();
 
-            expect(res.result).toStrictEqual(new Uint8Array(Buffer.from(bs.substring(2), "hex")));
+            expect(res).toStrictEqual(Buffer.from(bs.substring(2), "hex"));
 
             if (bs.length <= 4 || Math.random() >= 0.5) {
                 let val = ((Math.random() * 256) | 0);
 
-                await contract.functions.push(new Uint8Array([val]));
+                await program.methods.push([val]).accounts({ dataAccount: storage.publicKey }).rpc();
 
                 let valStr = val.toString(16);
                 valStr = valStr.length == 1 ? "0" + valStr : valStr;
 
                 bs += valStr;
             } else {
-                res = await contract.functions.pop();
+                await program.methods.pop().accounts({ dataAccount: storage.publicKey }).rpc();
 
                 let last = bs.slice(-2);
 
-                expect(res.result).toStrictEqual(
-                    new Uint8Array(Buffer.from(last, "hex")));
+                // TODO: rpc calls cannot return anything; this should be fixed in Anchor and
+                // this can be uncommented.
```

Comment:
```suggestion
                // TODO: rpc calls cannot return anything; this should be fixed in Anchor, so that
                // this line can be uncommented.
```